docs/next/guides/getting-started/installation.md

```diff
@@ -41,41 +86,86 @@ Run the following command in your terminal
   ```
 
   </code-block>
-  <code-block title="Nuget">
+  <code-block title="NuGet">
 
   ```bash
   PM> Install-Package Handsontable
   ```
 
   </code-block>
-  <code-block title="CDN">
+</code-group>
 
-  ```html
-  // minified JS
-  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.js"></script>
+### Using a CDN
 
-  // minified CSS
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.css" />
-  ```
+To get Handsontable's files from a CDN, use the following locations:
 
-  </code-block>
-</code-group>
+```js
+https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.js
 
-## Create a placeholder
+https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.css
+```
```

Comment:
I think that a list of links would actually be the most useful here:

- [https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.js](https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.js)
- [https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.css](https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.css)